doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,305 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "aioConnectorMetadataSchemaVersion": "1.0.0",
+  "definitions": {
+    "additionalConfigurationType": {
+      "type": "object",
+      "description": "The schema for this 'additional configuration' field. Must provide inline schema xor the file that contains the schema",
```

Comment:
>xor [](http://example.com/codeflow?start=103&length=3)

typo

Reply:
This is deliberate. A user can specify one or the other, but not both

Reply:
hmm. i did not get it. the text says "xor" this is what i meant by typo. it should be "or"